src/util.js

```diff
@@ -0,0 +1,13 @@
+'use strict'
+
+const randomBytes = require('random-bytes')
```

Comment:
hmm why do you need a module for random bytes? there is `crypto.getRandomValues()` in the browser and `crypto.randomBytes()` in node.js